shell/server/pre-meteor.js

```diff
@@ -210,6 +210,25 @@ function checkMagic(buf, magic) {
   return true;
 }
 
+function serveSelfTest(req, res) {
+  inMeteor(() => {
```

Comment:
Upon re-reading this comment, I see that `inMeteor()` is something that we wrote, not something that comes with Meteor. Since `try/except` is all I need, I'll switch to that.